examples/core/source.py

```diff
@@ -14,132 +18,133 @@
+# > to point to the assets folder.
 
-# The only way to create a source, is to create it from a project.
+if USE_DOCKER:  # Running on the remote server.
+    assets_data_path = Path("/app") / "assets"
+else:
+    assets_data_path = Path("/path/to/your/download/assets/directory")
 
-# +
-p = Project(speos=speos)
-print(p)
-# -
+# ### Connect to the RPC Server
+# This Python client connects to a server where the Speos engine
+# is running as a service. In this example, the server and
+# client are the same
+# machine.
 
-# ## Create
+speos = Speos(host=HOSTNAME, port=GRPC_PORT)
 
-# Create locally
+# ### Create a new project
+#
+# The only way to create a source using the core layer, is to create it from a project.
+# The ``Project`` class is instantiated by passing a ``Speos`` instance
 
-# The mention "local: " is added when printing the source.
+p = Project(speos=speos)
+print(p)
 
-# +
-intensity_file_path = str(assets_data_path / "IES_C_DETECTOR.ies")
 
+# ### Source Creation
+#
+# **Create locally:**
+# The mention "local: " is added when printing the source data and information is not yet
+# pushed to the RPC server
+
+intensity_file_path = str(assets_data_path / IES)
 source1 = p.create_source(name="Luminaire.1", feature_type=SourceLuminaire)  # type luminaire
 source1.set_intensity_file_uri(uri=intensity_file_path)
 print(source1)
-# -
-
-# ## Push it to the server.
 
-# Now that it is committed to the server, the mention "local: " is no more present when printing the
+# **Push it to the server.**
+#
+# After it is committed to the server, the mention "local: " is no more present when printing the
 # source.
 
-# +
 source1.commit()
 print(source1)
-# -
-
-# ## Another example
 
+# **Changing additional Source Properties**
+#
 # Setting several more characteristics.
 
-# +
-intensity_file_path = str(assets_data_path / "IES_C_DETECTOR.ies")
 
+intensity_file_path = str(assets_data_path / IES)
 source2 = p.create_source(name="Luminaire.2", feature_type=SourceLuminaire)
 source2.set_intensity_file_uri(uri=intensity_file_path)
 source2.set_flux_radiant()  # select flux radiant with default value
 # choose the source location [Origin, Xvector, Yvector, Zvector]
 source2.set_axis_system(axis_system=[20, 50, 10, 1, 0, 0, 0, 1, 0, 0, 0, 1])
-source2.set_spectrum().set_blackbody()  # choose blacbody with default value for the source spectrum
+source2.set_spectrum().set_blackbody()  # choose blackbody with default value for the spectrum
 source2.commit()  # Push to the server
 print(source2)
-# -
-
-# ## Read
-
-# ### Source Instance
 
+# **Source Instance**
+#
 # As mention "local: " is added if it is not yet committed to the server.
 
-# +
 print(source1)
-# -
-
-# ### Project
 
+# **Project:**
+#
 # Committed feature will appear inside the project information.
 
-# +
 print(p)
-# -
-
-# ## Update
 
+# **Update:*
+#
 # Tipp: if you are manipulating a source already committed, don't forget to commit your changes.
```

Comment:
Also, do you want to put the Tip content also as Note content? If not, that is ok.